pkg/client/rootd/dns/server_darwin.go

```diff
@@ -50,7 +50,8 @@ func (r *resolveFile) write(fileName string) error {
 // or, if not on a Mac, follow this link: https://www.manpagez.com/man/5/resolver/
 func (s *Server) Worker(c context.Context, dev vif.Device, configureDNS func(net.IP, *net.UDPAddr)) error {
 	resolverDirName := filepath.Join("/etc", "resolver")
-	resolverFileName := filepath.Join(resolverDirName, "telepresence.local")
+	localResolverFileName := filepath.Join(resolverDirName, "telepresence.local")
+	tel2SearchResolverFileName := filepath.Join(resolverDirName, "tel2-search.local")
```

Comment:
This suggests a domain named `tel2-search`. I don't think we want that. The `tel2-search` is intended as a search path only. It's not a domain.